build.gradle

```diff
@@ -44,21 +32,21 @@ publishing {
         mavenJava(MavenPublication) {
             groupId = ARTIFACT_PACKAGE
             artifactId = ARTIFACT_NAME
-            version = ARTIFACT_VERSION
             from components.java
-        }
-    }
-    repositories {
-        maven {
-            url = "https://maven.pkg.github.com/${COMPANY}/${REPOSITORY_NAME}"
-            credentials {
-                username = System.getenv("GITHUB_USERNAME")
-                password = System.getenv("GITHUB_TOKEN")
+            pom {
```

Comment:
In order to be able to enforce a hard gate on what can and can't be published to the jfrog repo, we don't publish jars directly from our gradle/maven configs.  Instead, we build the jars and declare a `BUILD_ARTIFACTS_JAVA` path to describe them.  

This lets the build system run all the compliance checks required for a Java artifact, then if they pass it will publish the jar on our behalf.  If the compliance checks fail, it will fail the build and describe what needs to be changed.  

https://github.com/Workiva/flip-tables/pull/4/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R21